homeassistant/components/vesync/fan.py

```diff
@@ -191,7 +191,7 @@ async def async_set_percentage(self, percentage: int) -> None:
             percentage_to_ordered_list_item(self.device.fan_levels, percentage)
         ):
             raise HomeAssistantError(
-                "An error occurred while changing fan speed:"
+                "An error occurred while changing fan speed: "
```

Comment:
I see in other items removing the context of the sentence and just putting the response message.   Is that intentional to have this one different?

Reply:
Intentional, thought it would be helpful distinguish between the different actions within `async_set_percentage` (Turn on/off, set speed, manual mode)